exercises/arrays/secondLargestNumber/index.ts

```diff
@@ -0,0 +1,7 @@
+const arrTask3: number[] = [25, 143, 89, 13, 105];
+
+const secondLargest = (arrTask3: number[]): number => {
+  return arrTask3.sort((a, b) => a - b)[arrTask3.length - 2];
```

Comment:
Do we relly need to pass `(a, b) => a - b)` here?

Reply:
I believe yes, if only `sort()` is used, we will get the following sort result: [105, 13, 143, 25, 89] (because array elements are sorted as strings by default).
Whereas with `sort((a, b) => a - b)` we have [13, 25, 89, 105, 143].